presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -330,7 +330,7 @@ else if (serverConfig.isCatalogServer()) {
             install(new CatalogServerModule());
         }
         else if (serverConfig.isCoordinator()) {
-            install(new CoordinatorModule());
+            install(new CoordinatorModule(serverConfig.isWebUIEnabled()));
```

Comment:
We can bind `ServerConfig` in `CoordinatorModule` and check for UI config there?

```
 ServerConfig severConfig = buildConfigObject(ServerConfig.class);
```